server/api/brands.js

```diff
@@ -2,16 +2,62 @@ const router = require('express').Router()
 const {Drink, Brand} = require('../db/models')
 module.exports = router
 
-router.get('/', (req, res, next) => {
-  Brand.findAll({include: [Drink]})
-    .then(users => res.json(users))
-    .catch(next)
+router.get('/', async (req, res, next) => {
+  try {
+    const brands = await Brand.findAll({include: [Drink]})
+    res.json(users)
+  }
+  catch (err){
+    next(err)
+  }
 })
 
-router.get('/:brandId', (req, res, next) => {
-  Drink.findAll({
-    where: {brandId: req.params.brandId},
-  })
-    .then(users => res.json(users))
-    .catch(next)
+router.get('/:brandId', async (req, res, next) => {
+  try {
+    const brand = await Brand.findbyId(req.params.brandId)
+    res.json(brand)
+  }
+  catch (err){
+    next(err)
+  }
+})
+
+router.post('/', async (req, res, next) => {
+  try {
+    const [brand] = await Brand.findOrCreate({
+      where: {
+        id: req.body.id
+      }
+    })
+    res.json(brand)
+  }
+  catch (err){
+    next(err)
+  }
+})
+
+router.put('/:brandId', async (req, res, next) => {
+  try {
+    const brand = Brand.update(req.body, {
+      where: {
+        id: req.params.brandId
+      }
+    })
+    res.send(brand)
```

Comment:
if you update the class you need returning:true
also deconstruct to get the updated instance